components/HomePage/Price.tsx

```diff
@@ -64,7 +65,7 @@ const PriceCard = styled.div`
   transition: all 0.2s ease-in-out;
 
   &:hover {
-    transform: scale(1.1);
+    transform: scale(1.05);
     transition: all 0.2s ease-in-out;
```

Comment:
No need to put this property in the `hover` state. You already have it in the regular state. That is enough